sources/meta-ornl/conf/machine/include/ornl.inc

```diff
@@ -24,6 +24,7 @@ UBI_VOLNAME  = "rootfs"
 
 # Variscite BSP default providers
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-variscite"
+# FIXME: this breaks thud when it uses 4.9.88
```

Comment:
Does any of our projects still use Thud?  Can we EOL Thud just as the official Yocto Project has?